### DISCUSSION TITLE
Quick Docs Feedback (new user)

### DISCUSSION THREAD

**Discussion by fergusmeiklejohn** [tailwindlabs/tailwindcss#2498] [Ideas]
I think the Installation page should include information from two other pages, which I found I needed to get Tailwind to work for me: https://tailwindcss.com/docs/installation

- Setting up purge CSS https://tailwindcss.com/docs/controlling-file-size#removing-unused-css

- How to change your font, from the TailwindUI docs https://tailwindui.com/documentation#optional-add-the-inter-font-family

Also is there any value in using `@tailwind base;` etc instead of `@import "tailwindcss/base";` etc?  Because the former throws confusing linting errors https://tailwindcss.com/docs/installation#add-tailwind-to-your-css
I enjoyed watching Adam's talk at laracon last year in which he recommends the latter import syntax so maybe that should be the default? https://youtu.be/J_7_mnFSLDg?t=685

**Comment by simonswiss** [comment]
Hey @fergusmeiklejohn, 

Thank you for taking the time to provide feedback on your experience "onboarding" Tailwind CSS as a new user. This is very valuable! 👍

We're in the process of doing a docs website refresh in anticipation for the Tailwind 2.0 release, and we'll keep these suggestions in mind!

I hope you enjoy using Tailwind. Let us know if you have questions or want to share what you've been building with it! 🎉
